src/utils/teamScraper.py

```diff
@@ -0,0 +1,190 @@
+import time
+import json
+import requests
+import shutil
+import os
+from collections import defaultdict
+from dotenv import load_dotenv
+
+load_dotenv()
+
+database_id = os.environ.get('NOTION_DB_ID')
+url = f"https://api.notion.com/v1/databases/{database_id}/query"
+headers = {
+    "accept": "application/json",
+    "Notion-Version": "2022-06-28",
+    "Authorization": os.environ.get('NOTION_TOKEN')
+}
+query = {
+    "filter": {
+        "property": "Status",
+        "select": {
+            "equals": "Current Member"
+        }
+    }
+}
+
+eboard_roles = set(["Executive Director", "Technical Director", "UX Director",
+                    "Operations Director", "Marketing & Events Director", "E-Board Liaison"])
+eboard_roles_order = {"Executive Director": 0, "Technical Director": 1, "UX Director": 2,
+                      "Operations Director": 3, "Marketing & Events Director": 4, "E-Board Liaison": 5}
+head_of_roles = set(["Head of Recruiting", "Head of DX", "Head of UX",
+                     "Head of Project Acquisition", "Head of Community"])
+head_of_roles_order = {"Head of Recruiting": 0, "Head of DX": 1, "Head of UX": 2,
+                       "Head of Project Acquisition": 3, "Head of Community": 4}
+brand_roles_order = {"Marketing & Events Director": 0, "Brand Designer": 1}
+project_teams = set(["Carpool", "Faculty Activity Tracker", "GraduateNU",
+                    "HappyEastie", "MFA", "ScoutTrek", "SearchNEU", "SGA Tooling"])
+project_team_member_order = {"Project Lead": 0,
+                             "Design Lead": 1, "Designer": 2, "Developer": 3}
+team_order = ["E-Board", "Head Ofs", "Brand", "Carpool",
+              "Faculty Activity Tracker", "GraduateNU", "HappyEastie", "MFA", "ScoutTrek", "SearchNEU", "SGA Tooling"]
+output_file = "../content/team/team.json"
+
+
+def get_current_members_from_notion():
+    print("Retrieving current members from Notion Club Directory")
+    response = requests.post(url, headers=headers, json=query)
+    data = response.json()
+    results = data["results"]
+    return results
+
+
+def make_person(name, team_name, role, linkedin, portfolio, email):
+    social_media = {
+        "email": email or None,
+        "linkedIn": linkedin or None,
+        "portfolio": portfolio or None
+    }
+    social_media_filtered = {k: v for k,
+                             v in social_media.items() if v is not None}
+
+    return {
+        "name": name,
+        "team": {
+            "name": team_name,
+            "role": role,
+        },
+        "profileImage": f"./profileImages/SP23/{name}.png",
+        "socialMedia": social_media_filtered
+    }
+
+
+def generate_team_mappings(roles, teams):
```

Comment:
Somewhere in here we need to account for Tech Leads. Not sure whether we should just remove Developer role from them in the Notion or if you want to deal with parsing the multiple role case here, but right now both SGA and ScoutTrek are missing the acknowledgement of Chan Mi and Ross respectively.

Reply:
removed Developer role from both Ross and Chan Mi on Notion and re-updated the team JSON file